programmers/난이도별/level01.시저_암호/sjjyy.java

```diff
@@ -0,0 +1,30 @@
+package 난이도별.level01.시저_암호;
+
+public class sjjyy
+{
+    public static String solution(String s, int n) {
+
+        StringBuilder ans = new StringBuilder();
+
+        for (int i = 0 ; i < s.length() ; i++) {
+            char c = s.charAt(i);
+
+            if (Character.isLowerCase(c))
+                c = (char)('a' + (c + n - 'a') % 26);
+
+                else if (Character.isUpperCase(c))
+                    c = (char) ('A' + (c + n - 'A') % 26);
+
+                    ans.append(c);
```

Comment:
```suggestion
            ans.append(
                Character.isLowerCase(c) ? (char)('a' + (c + n - 'a') % 26) :
                Character.isUpperCase(c) ? (char)('A' + (c + n - 'A') % 26) :
                c
            );
```

이렇게 표현할 수 있을 것 같아요!

사실 그냥 제가 해보고 싶은 실험적 표현이랄까..  😓